src/core.js

```diff
@@ -104,8 +104,11 @@ async function sync(cloneUrl, localDirectory, {ref, checkout}) {
   }
 
   await git.cwd(localDirectory);
-  log.info(`> Fetching ${ref}...`);
-  await git.fetch('origin', ref);
+  const remoteName = cloneName || 'origin';
+  await git.removeRemote(remoteName).catch((error) => {}); // eslint-disable-line no-unused-vars
```

Comment:
this cloneUrl is specific to the github-token so if a token is revoked, then all the old remotes with invalid tokens are still there and will fail on the fetch.

always doing a `.removeRemote()` right here means the cloneUrl provided is always used. 

`removeRemote` caught because we don't care if it's removing something that isn't there.